src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -489,9 +489,9 @@ void PolicyHandler::SendOnAppPermissionsChanged(
 }
 
 void PolicyHandler::OnPTExchangeNeeded() {
+  LOG4CXX_AUTO_TRACE(logger_);
   POLICY_LIB_CHECK_VOID();
-  MessageHelper::SendOnStatusUpdate(policy_manager_->ForcePTExchange(),
-                                    application_manager_);
+  policy_manager_->ForcePTExchange();
```

Comment:
@okozlovlux I am perfectly fine with your explanation, but I am concerned about potential functional regressions, so I would like to be sure that you are aware of the impact of this change.